libevm/LegacyVM.cpp

```diff
@@ -97,10 +97,13 @@ void LegacyVM::adjustStack(unsigned _removed, unsigned _added)
 
 void LegacyVM::updateSSGas()
 {
+    if (m_schedule->sstoreThrowsIfGasBelowCallStipend() && m_io_gas <= m_schedule->callStipend)
```

Comment:
Why do we only throw OOG if EIP2200 is enabled, I'd expect us to always throw OOG if we don't have enough gas to cover the call stipend?

Reply:
This rule is introduced only in EIP-2200.
(without it we normally throw in case of OOG from the `updateIOGas` call here https://github.com/ethereum/aleth/blob/0faec9d18e09459462f392d710b1c0d7f5008d5f/libevm/LegacyVM.cpp#L1678)